x/mongo/driver/operation.go

```diff
@@ -1078,12 +1075,6 @@ func (op Operation) createWireMessage(
 	maxTimeMS uint64,
 	conn Connection,
 ) ([]byte, startedInformation, error) {
-	// If topology is not LoadBalanced, API version is not declared, and wire version is unknown
-	// or less than 6, use OP_QUERY. Otherwise, use OP_MSG.
-	if desc.Kind != description.LoadBalanced && op.ServerAPI == nil &&
-		(desc.WireVersion == nil || desc.WireVersion.Max < wiremessage.OpmsgWireVersion) {
-		return op.createQueryWireMessage(maxTimeMS, dst, desc)
-	}
 	return op.createMsgWireMessage(ctx, maxTimeMS, dst, desc, conn)
```

Comment:
We can remove "createWireMessage" and use "createMsgWireMessage" instead.

Reply:
Great idea!